partials/tabs/about.html

```diff
@@ -129,4 +129,4 @@ <h2 class="fine center">about</h2>
   $('ul.tabs').tabs();
 });
 
-</script>
+</script>
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp8v872ato/partials/tabs/about.html
+++ b/tmp/tmp8v872ato/partials/tabs/about.html
@@ -129,4 +129,4 @@
   $('ul.tabs').tabs();
 });
 
-</script>+</script>
```